### ISSUE TITLE
[Bug] module 'nvim-web-devicons' not found

### ISSUE THREAD

**Issue by mcchrish** [famiu/feline.nvim#78]
**Neovim version**
```
NVIM v0.5.0
Build type: Release
LuaJIT 2.1.0-beta3
Compilation: clang -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1 -DNVIM_TS_HAS_SET_MATCH_LIMIT -O2 -DNDEBUG -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wshadow -Wconversion -Wmissing-prototypes -Wimplicit-fallthrough -Wvla -fstack-protector-strong -fno-common -fdiagnostics-color=auto -DINCLUDE_GENERATED_DECLARATIONS -D_GNU_SOURCE -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -DMIN_LOG_LEVEL=3 -I/tmp/neovim-20210702-14987-rep9di/neovim-0.5.0/build/config -I/tmp/neovim-20210702-14987-rep9di/neovim-0.5.0/src -I/usr/local/include -I/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include -I/usr/local/opt/gettext/include -I/tmp/neovim-20210702-14987-rep9di/neovim-0.5.0/build/src/nvim/auto -I/tmp/neovim-20210702-14987-rep9di/neovim-0.5.0/build/include
Compiled by brew@BigSur

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/usr/local/Cellar/neovim/0.5.0/share/nvim"
```

**Feline version**
Current master.

**Describe the bug**
```
E5108: Error executing lua ...k/packer/start/feline.nvim/lua/feline/providers/file.lua:97: module 'nvim-web-devicons' not found:
        no field package.preload['nvim-web-devicons']No cache entry
```

**To Reproduce**

I don't have web-devicons installed. Just use the basic `setup {}`.

**Expected behavior**

Should work without devicons installed since it's optional.

**Comment by famiu** [comment]
Duplicate of #77. Fix is already on latest develop. Will be in next patch release (which is coming out after #76 is merged). Guess I'll keep the issue open until that's done

**Comment by famiu** [comment]
Alright @mcchrish, v0.1.1 is out and your issue should be fixed now
